inst/extdata/installPackages.R

```diff
@@ -34,18 +60,59 @@ installPackages <- function(pkgs, lib, repos = getOption("repos")) {
   }
 }
 
-lib <- Sys.getenv("R_LIBS")
-if (nchar(lib) == 0) {
-  stop("Could not get a value for R_LIBS environment variable")
+nodename <- Sys.info()["nodename"]
+
+# if this is our Very Special development machine
+if (nodename == "IGSWZTWWWSASHA") {
+  lib <- .libPaths()[1]
+} else {
+  # it's the general public
+  lib <- Sys.getenv("R_LIBS")
+  
+  if (nchar(lib) == 0) {
+    stop("Could not get a value for R_LIBS environment variable")
+  }
 }
 
-# all packages except devtools and its prerequisites are held back to older
-# versions
-installPackages(pkgs, lib, "https://mran.microsoft.com/snapshot/2016-03-31")
+pkgs <-
+  grep(
+    "gsplot", repgenImports(lib.loc = lib),
+    value = TRUE, fixed = TRUE, invert = TRUE
+  )
+
+# all packages are held back to older, MRAN versions
+installPackages(c(pkgs, "devtools"), lib, "https://mran.microsoft.com/snapshot/2016-03-31")
+
+if (nodename == "IGSWZTWWWSASHA") {
+  # To be able to install gsplot from GitHub using devtools (below), download 
+  # DOI root certificate and append to openssl package's certificate bundle. See
+  # also https://usgs-cida.slack.com/archives/r-activities/p1469472383000332
+  
+  # locate & read the openssl certificate bundle
+  cert_bundle <- httr:::find_cert_bundle()
+  cert_bundle_lines <- readLines(cert_bundle)
+  
+  # check to see if DOI root certificate is already in the certificate bundle...
+  if (!any(grepl("DOI Root CA", cert_bundle_lines, fixed = TRUE))) {
+    # ...so that we only download & append the DOI root certificate once
+    
+    doiRootCA <- tempfile("DOIRootCA.")
+    download.file("http://blockpage.doi.gov/images/DOIRootCA.crt", doiRootCA)
+    
+    cat(
+      c("", "DOI Root CA", "===================="),
+      file = cert_bundle,
+      sep = "\n",
+      append = TRUE
+    )
+    file.append(cert_bundle, doiRootCA)
+    file.remove(doiRootCA)
+  }
+}
 
-# devtools is only needed to install gsplot and repgen from source on GitHub;
-# it's not used by repgen in production
-installPackages(c("devtools"), lib, "https://cloud.r-project.org")
+# gsplot hasn't made it to an offical repository yet, so it needs to be
+# installed with devtools
+devtools::install_github("USGS-R/gsplot#420")
```

Comment:
Once GRAN has the new version of gsplot (0.7.1, aka PR #420) we could install from there instead of with devtools, but this works anyway.